src/test/kotlin/model/DealerPlayerTest.kt

```diff
@@ -0,0 +1,25 @@
+package model
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+class DealerPlayerTest {
+    @Test
+    @DisplayName("dealer 21점 이상이면 lose 를 반환한다")
+    fun `over21ScoreLose`() {
+        val dealer = DealerPlayer()
+        val player = Player("hello")
+        val cards = listOf(Card(Suit.CLUBS, Denomination.ACE), Card(Suit.CLUBS, Denomination.QUEEN), Card(Suit.CLUBS, Denomination.QUEEN), Card(Suit.CLUBS, Denomination.QUEEN))
+        val cardDealer = listOf(Card(Suit.CLUBS, Denomination.ACE), Card(Suit.CLUBS, Denomination.QUEEN), Card(Suit.CLUBS, Denomination.QUEEN), Card(Suit.CLUBS, Denomination.QUEEN))
+        receiveCard(dealer, cardDealer)
+        receiveCard(player, cards)
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
        val dealer = DealerPlayer(
            listOf(
                Card(Suit.CLUBS, Denomination.ACE),
                Card(Suit.CLUBS, Denomination.QUEEN),
                Card(Suit.CLUBS, Denomination.QUEEN),
                Card(Suit.CLUBS, Denomination.QUEEN)
            )
        )
        val player = Player(
            "hello",
            listOf(
                Card(Suit.CLUBS, Denomination.ACE),
                Card(Suit.CLUBS, Denomination.QUEEN),
                Card(Suit.CLUBS, Denomination.QUEEN),
                Card(Suit.CLUBS, Denomination.QUEEN)
            )
        )
```

Reply:
abstract 에서 val 로 설정되어 있어서 card list 를 set 을 할 수 없는데요. var 로 바꾸는건 생각하신걸까요?